File: Feitos em C/2516.c
#include <stdio.h>
int main() {
	int s;
	while(scanf("%d",&s)==1){
		int va,vb;
		double temp=0;
		scanf("%d",&va);
		scanf("%d",&vb);
		if(va>vb){
			temp=(double)s/(va-vb);
			printf("%.2f\n",temp);
		}
		else{
			printf("impossivel\n");
		}
	}
	return 0;
}

File: Feitos em Java/2516.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        while(leitor.hasNext()){
            double s = leitor.nextDouble();
            int va = leitor.nextInt();
            int vb = leitor.nextInt();
            double temp=0;
            if(va>vb){
                temp=s/(va-vb);
                System.out.printf("%.2f\n",temp);
            }
            else{
                System.out.println("impossivel");
            }
	}
    }
}